tests/nd_range/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required (VERSION 3.0) # The minimum version of CMake necessary to build this project
+project (nd_range) # The name of our project
+
+declare_trisycl_test(TARGET nd_range TEST_REGEX
+" 6 4 8
+ 3 2 4
+ 0 0 0
+ 6 4 8
+ 3 2 4
+ 0 0 0
+ 2 2 2
+ 2 2 2
+ 4
+ 2
+ 3
+ 10 20
+ 2 5
+ 0 0
+ 1
+ 6 4
+ 6 5")
+ 
```

Comment:
Same issue : now we have a new line with 1 trailing space without EOL. :-)
On Windows, NotePad++ has a Show/Symbols menu and also Edit/Blank operations menu that might be useful.